### ISSUE TITLE
Question about retrieving the corpus

### ISSUE THREAD

**Issue by 421zuoduan** [TideDra/zotero-arxiv-daily#27]
Hi,

Thank you for your amazing work. I'm trying to use the LLM API for testing, and I encountered the following error when retrieving the corpus:

```
Run uv run main.py --debug
Using CPython 3.***.***0
Creating virtual environment at: .venv
Installed 64 packages in 298ms
20[24](https://github.com/421zuoduan/zotero-arxiv-daily/actions/runs/12293408518/job/34307078823#step:4:25)-***2-***2 09:45:3***.546 | **mDEBUG    | __main__:<module>:258 - **mDebug mode is on.
2024-***2-***2 09:45:3***.546 | **mINFO     | __main__:<module>:26*** - **mRetrieving Zotero corpus...
Traceback (most recent call last):
  File "/home/runner/work/zotero-arxiv-daily/zotero-arxiv-daily/main.py", line [26](https://github.com/421zuoduan/zotero-arxiv-daily/actions/runs/12293408518/job/34307078823#step:4:27)2, in <module>
    corpus = get_zotero_corpus(args.zotero_id, args.zotero_key)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/zotero-arxiv-daily/zotero-arxiv-daily/main.py", line 34, in get_zotero_corpus
    paths = [get_collection_path(col) for col in c['data']['collections']]
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/zotero-arxiv-daily/zotero-arxiv-daily/main.py", line 34, in <listcomp>
    paths = [get_collection_path(col) for col in c['data']['collections']]
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/zotero-arxiv-daily/zotero-arxiv-daily/main.py", line [29](https://github.com/421zuoduan/zotero-arxiv-daily/actions/runs/12293408518/job/34307078823#step:4:30), in get_collection_path
    if p := collections[col_key]['data']['parentCollection']:
            ~~~~~~~~~~~^^^^^^^^^
KeyError: 'MTZYSDA6'
Error: Process completed with exit code ***.
```

It seems to be an issue with the `ZOTERO_ID` and `ZOTERO_KEY` settings. I have correctly set the `ZOTERO_ID` to an eight-digit number, and the zotero website shows that the API is being called. However, the error persists. 

Could you please help me understand what might be causing this issue? Thank you in advance for your assistance.

**Comment by TideDra** [comment]
hi， it seems some bugs exist in the resolve of paper path. how many collections are there in your zotero library?

**Comment by 421zuoduan** [comment]
> hi， it seems some bugs exist in the resolve of paper path. how many collections are there in your zotero library?

I previously had around a dozen repositories in my Zotero library. I just cleaned up some of them, leaving only nine repositories. I retested the process, and the same error still occurred, but with a slightly different message. The error now is `KeyError: 'DIEZD9QK'` instead of `KeyError: 'MTZYSDA6'`. Additionally, the file names in my Zotero library include both Chinese and English characters. I am using Zotero 7 but not 6
